pallets/valve/src/lib.rs

```diff
@@ -46,18 +49,20 @@ pub mod pallet {
 		traits::{Contains, PalletInfoAccess},
 	};
 	use frame_system::pallet_prelude::*;
-	use pallet_automation_time::{self as automation_time};
 	use sp_std::vec::Vec;
 
 	#[pallet::config]
-	pub trait Config: frame_system::Config + pallet_automation_time::Config {
+	pub trait Config: frame_system::Config {
 		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
 
 		/// Weight information for the extrinsics in this module.
 		type WeightInfo: WeightInfo;
 
 		/// The pallets that we want to close on demand.
 		type ClosedCallFilter: Contains<Self::Call>;
+
+		/// The AutomationTime pallet.
+		type AutomationTime: Shutdown;
```

Comment:
The valve pallet makes many assumptions about the automation time pallet. I've only unraveled some of those assumptions here (events and extrinsics are still about tasks start/stopping, etc) so for now I've tried to make it clear that this type is expecting the automation time pallet.

It's not great but I think it's still a step in the right direction of decoupling these pallets completely.